framework/base/Model.php

```diff
@@ -601,26 +636,32 @@ public function getFirstError($attribute)
     }
 
     /**
-     * Adds a new error to the specified attribute.
-     * @param string $attribute attribute name
+     * Adds a new error to the specified attribute or common error.
+     * @param string $attribute attribute name. Leave `null` for common errors (not related with particular attribute).
      * @param string $error new error message
+     * @throws InvalidParamException in case of attribute not exist
      */
-    public function addError($attribute, $error = '')
+    public function addError($attribute = null, $error = '')
     {
+        if ($attribute === null) {
```

Comment:
Since when you're getting all errors, there's `*` key. It's natural that one would think setting `*` will add errors to that group.